nautobot/core/jobs/__init__.py

```diff
@@ -178,9 +178,12 @@ def run(self, *, content_type, query_string="", export_format="csv", export_temp
             "sort",
             "table_changes_pending",
         )
-        filter_params = get_filterable_params_from_filter_params(
-            query_params, default_non_filter_params, filterset_class()
-        )
+        if "saved_view" in query_params:
+            filter_params = SavedView.objects.get(pk=query_params["saved_view"]).config.get("filter_params", {})
+        else:
+            filter_params = get_filterable_params_from_filter_params(
+                query_params, default_non_filter_params, filterset_class()
+            )
```

Comment:
It's possible to apply additional filters via query params "in addition to" a Saved View's filters, so this probably shouldn't be an `else` case but something more like: 

```suggestion
        filter_params = {}
        if "saved_view" in query_params:
            filter_params.update(SavedView.objects.get(pk=query_params["saved_view"]).config.get("filter_params", {}))
        filter_params.update(get_filterable_params_from_filter_params(
            query_params, default_non_filter_params, filterset_class()
        )
```

Let's please make sure to augment the testing for this job to cover this scenario.

Reply:
Probably it is possible, but as I understand if we have `saved_view` then we shouldn't have more query_params so why it's needed ?

Additionally, I covered this case in testing.

Reply:
I did a quick check on the demo. If we will add some filters on top of saved view filters, nautobot will redirect us to the url with both: query params from saved view and new filter added. 

So we have couple possible scenarios:
- Just query params: `/?parent=AMER&status=Active`
- Just saved view: `/?saved_view=314cd0db-9a3b-436a-a68f-7984c0b7c4bb`
- Both - for tests I created a saved view with `parent=AMER` and clicked `status=Active` and was redirected to: `/?saved_view=314cd0db-9a3b-436a-a68f-7984c0b7c4bb&parent=AMER&status=Active`
- Saved view, clicked status filter, removed built-in parent filter: `/?saved_view=314cd0db-9a3b-436a-a68f-7984c0b7c4bb&status=Active`

I think that Daria solution mimics the "frontend" way of working. In approach with merging filters from query params and saved view, we will fail in the fourth case from above list.

Reply:
After double checking we should have something like `if "saved_view" in query_params and len(query_params) == 1` because if user will add some more filters, I believe we should focus on those "custom" one.